src/SignalR/common/Http.Connections/src/Internal/HttpConnectionsMetrics.cs

```diff
@@ -9,62 +9,40 @@ namespace Microsoft.AspNetCore.Http.Connections.Internal;
 
 internal readonly struct MetricsContext
 {
-    public MetricsContext(bool currentConnectionsCounterEnabled, bool connectionDurationEnabled, bool currentTransportsCounterEnabled)
+    public MetricsContext(bool connectionDurationEnabled, bool currentNegotiatedConnectionsCounterEnabled)
     {
-        CurrentConnectionsCounterEnabled = currentConnectionsCounterEnabled;
         ConnectionDurationEnabled = connectionDurationEnabled;
-        CurrentTransportsCounterEnabled = currentTransportsCounterEnabled;
+        CurrentNegotiatedConnectionsCounterEnabled = currentNegotiatedConnectionsCounterEnabled;
     }
 
-    public bool CurrentConnectionsCounterEnabled { get; }
     public bool ConnectionDurationEnabled { get; }
-    public bool CurrentTransportsCounterEnabled { get; }
+    public bool CurrentNegotiatedConnectionsCounterEnabled { get; }
 }
 
 internal sealed class HttpConnectionsMetrics : IDisposable
 {
     public const string MeterName = "Microsoft.AspNetCore.Http.Connections";
 
     private readonly Meter _meter;
-    private readonly UpDownCounter<long> _currentConnectionsCounter;
     private readonly Histogram<double> _connectionDuration;
-    private readonly UpDownCounter<long> _currentTransportsCounter;
+    private readonly UpDownCounter<long> _currentNegotiatedConnectionsCounter;
 
     public HttpConnectionsMetrics(IMeterFactory meterFactory)
     {
         _meter = meterFactory.Create(MeterName);
 
-        _currentConnectionsCounter = _meter.CreateUpDownCounter<long>(
-            "current-connections",
-            description: "Number of connections that are currently active on the server.");
-
         _connectionDuration = _meter.CreateHistogram<double>(
-            "connection-duration",
+            "signalr-http-transport-connection-duration",
             unit: "s",
             description: "The duration of connections on the server.");
 
-        _currentTransportsCounter = _meter.CreateUpDownCounter<long>(
-            "current-transports",
-            description: "Number of negotiated transports that are currently active on the server.");
-    }
-
-    public void ConnectionStart(in MetricsContext metricsContext)
-    {
-        // Tags must match connection end.
-        if (metricsContext.CurrentConnectionsCounterEnabled)
-        {
-            _currentConnectionsCounter.Add(1);
-        }
+        _currentNegotiatedConnectionsCounter = _meter.CreateUpDownCounter<long>(
+            "signalr-http-transport-current-negotiated-connections",
```

Comment:
I thought this was named `signalr-http-transport-current-connections` in API review.

Reply:
Yes.

I just sent an email to API review to recommend `current-negotiated-connections` as the name:

> When making this change, I found the new name has a couple of problems:
> 1. There's an existing current-connections event counter. The new proposed new signalr-http-transport-current-connections counter will start at a different time. That means the current-connections event counter and current-connections metric can have inconsistent values.
> 2. The proposed signalr-http-transport-current-connections is incremented when the connection has negotiated the transport and ends when the connection ends. That's a different time interval measured by signalr-http-transport-connection-duration. Someone could be confused why the times don't match.
> 
> I propose to rename what is currently "current-connections":
> * current-connections -> current-negotiated-connections

Reply:
I'm renaming it back to `signalr-http-transport-current-connections`. I'll make changes to get this name to "work" in a follow-up PR.